frontend/src/services/transactions/fill-manager.ts

```diff
@@ -30,13 +30,13 @@ export async function listenOnFulfillment(
 
   const timeoutPromise = new Promise((_resolve, reject) => {
     setTimeout(() => {
-      reject(new Error('Timeout!'));
+      reject(new Error('Timeout while waiting for fulfillment acknowledgement!'));
     }, eventListeningTimeout);
   });
 
   return Promise.race([fulfillmentPromise, timeoutPromise])
-    .then((fullRequestId) => {
-      return fullRequestId;
+    .then((fillRequestId) => {
```

Comment:
What's the point of this `then` call?

Reply:
Nothing! just the default `Promise` tree of `then-catch-finally` because I need `catch` and `finally` implementations! I removed it!